src/views/Scenario/Scenario.js

```diff
@@ -242,7 +242,7 @@ const Scenario = () => {
   const validationAreaDivider =
     userPermissionsOnCurrentScenario.includes(ACL_PERMISSIONS.SCENARIO.VALIDATE) ||
     currentScenarioData?.validationStatus !== SCENARIO_VALIDATION_STATUS.DRAFT ? (
-      <Divider orientation="vertical" flexItem />
+      <Divider orientation="vertical" flexItem sx={{ marginX: 2 }} />
```

Comment:
You can use `mx` who is little shorter but it is at you convenience...

Reply:
good idea, thanks ! :pray: